css/main.css

```diff
@@ -40,3 +38,19 @@ footer {
 footer svg {
   width: 1em;
 }
+
+#page-header {
+  color: white;
+  text-shadow: 4px 4px 1px rgb(13 66 163), -3px -3px 1px rgb(148 186 255);
+}
+
+/* screen reader friendly styling */
+.visually-hidden {
```

Comment:
This class is meant to be for text that is meant for screenreader users only, not for hiding elements in general. My idea was to only apply the visually-hidden class on the alt text for the fancy menu ("Altare leaps down and there are 5 cards for you to choose from") - that text is not needed for users that can see the screen

But maybe we should wait on setting up visually-hidden alt text until the end when the design is finalized, so I'd be fine with deleting this class and not worrying about getting accessibility perfect until the design is final